### DISCUSSION TITLE
The first load is slow.

### DISCUSSION THREAD

**Discussion by 573077632** [requarks/wiki#5351] [Help / Questions] [answered]
I use wiki.js in Chinese servers. Use QQ browser, Quark browser and Xiaomi's own browser. And the mobile terminal will be slow to open their own for the first time. The URL is https://cell.kin.wiki.

**Comment by 573077632** [comment, marked as answer]
I found the reason. Because I am in China, the related content of font4 and font5 websites can't be downloaded, which will cause the website to load the super card for the first time or get stuck.

**Reply by L-Jovi** [reply]
Hi, I have the same issue with you, but I try to check the font option as material instead of font4 or font5, it's still slowly, how could I fix it?
